src/backend/metal/src/command.rs

```diff
@@ -2302,6 +2471,10 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
         vertices: Range<VertexCount>,
         instances: Range<InstanceCount>,
     ) {
+        if instances.start >= instances.end {
```

Comment:
assert/debug_assert here?

Reply:
This is valid from Vulkan point of view (to have 0 instances). Or do you suggest catching the case of `instances.start > instances.end` with an assert? Given the amount of ranges we use, adding asserts to all of them looks like an overkill.

Reply:
I see, I though this was some sort of error handling. Didn't know that the spec allows that.

Reply:
In hindsight, we should just check for equality here. Negative ranges are going to panic automatically on our computation of the count (in debug, at least).